torch/csrc/dynamo/python_compiled_autograd.cpp

```diff
@@ -423,6 +423,7 @@ variable_list compiled_autograd(
             return ss.str();
           });
       saved.after(call.node->next_edges());
+      saved.debug_asserts();
```

Comment:
Can we add this above between the apply and the before on the next_edges to make sure the apply didn't leave the saved in a bad state?

Reply:
done